### DISCUSSION TITLE
[chrome] chrome.runtime is missing a method getContexts()

### DISCUSSION THREAD

**Discussion by sorokinvj** [DefinitelyTyped/DefinitelyTyped#68205] [Issues with a @types package] [answered]
from the [official documentation](https://developer.chrome.com/docs/extensions/reference/api/runtime#method-getContexts):

chrome.runtime.getContexts(
  filter: [ContextFilter](https://developer.chrome.com/docs/extensions/reference/api/runtime#type-ContextFilter),
  callback?: function,
)

Lets add this method, happy to contribute!

[a comment by an automated account is not shown]

**Comment by ghost** [comment]
#68195

**Reply by Semigradsky** [reply, marked as answer]
@sorokinvj please mark as answered
